lib/board.rb

```diff
@@ -0,0 +1,47 @@
+require "pry"
+
+class Board
+  attr_accessor :cells
+
+  def initialize
+    @cells = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ']
+  end
+
+  def reset!
+    @cells = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ']
+  end
+
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+  end
+
+  def position(n)
+    cells[n.to_i - 1]
+  end
+
+  def full?
+    !cells.include?(' ')
+  end
+
+  def turn_count
+    cells.select{|c| c == "X" || c == "O"}.length
+  end
+
+  def taken?(n)
```

Comment:
what is `n` ? Is it a number? Be descriptive of your argument and variable names. This makes your code much more readable.